prime-router/metadata/hl7_mapping/common/observation-result.yml

```diff
@@ -14,15 +14,17 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-2' ]
 
   - name: result-value-type-ce
-    condition: '%resource.value is CodeableConcept'
-    # Note that for v.2.5 and earlier CWE is not in-spec.
+    resource: '%resource.value'
+    condition: false
```

Comment:
This is super smart, in the short term this is a great work around for when we want to add an element for a receiver but can't do it at the root